### DISCUSSION TITLE
Canais e pessoas que vocês seguem/se inspiram!

### DISCUSSION THREAD

**Discussion by MathCunhaDev** [frontendbr/forum#879] [Discussão]
Eae pessoal! 

Queria saber mais sobre os programadores/desenvolvedores que vocês seguem e que me recomendassem alguns. Atualmente daqui sigo apenas o Medium do @woliveiras. 

Pode ser no Medium, Youtube ou aqui mesmo e etc. Ver a galera mais experiente é muito legal para mim ver os próximos passos a seguir e como seguir. 

Podem me recomendar outros canais que acharem interessantes também. ;D

Att.

**Comment by mariojvr** [comment]
Fala @MatCunha , tudo bem?

Segue a minha lista de pessoas que me influenciam, no  meu caso uso muito o Twitter:

PODCAST [devnaestrada](https://www.twitter.com/devnaestrada):

- Montanha - [femontanha](https://www.twitter.com/femontanha)
- Eduardo matos - [eduardojmatos](https://www.twitter.com/eduardojmatos)
- Ramon Sanches - [raymonsanches](https://www.twitter.com/raymonsanches)

Cursos Online AluraOnline:

- Flavio Almeida - [flaviohalmeida](https://www.twitter.com/flaviohalmeida) - [O cangaceiro javascript](http://cangaceirojavascript.com.br/) 
- Maurício Linhares - [mauriciojr](https://www.twitter.com/mauriciojr)
- Sérgio Lopes - [sergio_caelum](https://www.twitter.com/sergiocaelum)

Entre muitos outros. Aconselho, se já não os conhece, os podcasts [DEVNAESTRADA](https://devnaestrada.com.br/) e o [Hipters.tech](https://hipsters.tech/).

**Comment by andersoonweb** [comment]
Sigo bastante pessoas influente na área no Twiiter, mas vejo muita coisa que o [Addy Osmani](https://www.facebook.com/addyosmaniofficial/) e o [William](https://www.facebook.com/woliveiras) compartilham.

A maioria das pessoas que sigo é desse [guia](https://uptodate.frontendrescue.org/). No mais, você vai lendo os blogs e demais conteúdos e vai seguindo a galera, se não curtir só dar unfollow e construa suas próprias referências.

Aproveita e pega o links que essa galera disponibilizar com artigos, documentações, o que for, e salva no [feedly](https://feedly.com/), vai chegar uma hora que você vai ter bastante sites como referência, só abrir uma vez por dia, e dar uma olhada. Se o interesse é acompanhar mais de perto bastante pessoas, você vai conseguir dessa maneira. Espero ter ajudado @MatCunha.

**Comment by woliveiras** [comment]
Fico feliz por estar influenciando positivamente, galera. :heart: :heart: :heart: 

Eu sigo muita gente no Twitter, principalmente pessoas que me inspiram e compartilham muito conhecimento (técnico ou não): [aqui](https://twitter.com/w_oliveiras/following).

Também acompanho muita gente pelo feedly, [aqui tem meu feed](https://ufile.io/rn5mr). Não recomendo seguir todos os blogs que eu sigo por conta de serem muitos assuntos variados. Compensa mais olhar o .opml e subir pro seu feedly somente o que interessar.

**Comment by n3pu** [comment]
Acho q ainda não tenho 1 ano na comunidade, mas esses homão q tô de olho: @LFeh @fdaciuk @willianjusten. Tô conhecendo daqui mesmo e de cursos da Udemy. Já admiro pelo que fizeram e fazem. Medium e Twitter são bons canais pra encontrar tantos outros.

**Comment by isaacguerreir** [comment]
No youtube eu realmente encorajo as pessoas a verem o canal do Google Chrome Developers: https://www.youtube.com/user/ChromeDevelopers/
 
Em podcast eu aconselho o HTTP203 do pessoal do Google Chrome: https://developers.google.com/web/shows/http203/podcast/

No twitch eu realmente recomendo Sean Thomas Larkinn, ele é um dos colaboradores mais ativos do webpack e faz streamers muito bons: https://www.twitch.tv/thelarkinn

**Comment by raymonsanches** [comment]
Caraca, que honra ser citado aqui 🤗

**Comment by felipefialho** [comment]
Nois que voa @ewerto0n 👊

**Comment by femontanha** [comment]
Obrigado por nos citar, uma honra <3

**Comment by ninetails** [comment]
Sei que os caras aqui manjam pra caraleo, mas como n conheço muito a comunidade mando algumas inspirações que sigo:

Medium:
- [Eric Elliot](https://medium.com/@_ericelliott)
- [Kent C. Dodds](https://medium.com/@kentcdodds)
- [freeCodeCamp](https://medium.freecodecamp.org/)
- [Addy Osmani](https://medium.com/@addyosmani)

YT:
- [MPJ](https://www.youtube.com/channel/UCO1cgjhGzsSYb1rsB4bFe4Q)
- Vídeos de Conferências são sempre bem informativos pra saber o que tá rolando lá fora

Menção honrosa:
- [Dan Abramov](https://medium.com/@dan_abramov)

**Comment by klarkc** [comment]
DevNaEstrada (Montanha e Ramon), MPJ (Fun Fun Function).

Fiquei chateadasso porque descobri que passei pelo montanha na última BrasilJS e não falei com ele (não o reconheci, me falaram depois q ele estava lá) :(

**Comment by femontanha** [comment]
@klarkc que isso jovem! Bora trocar uma idéia :)

**Comment by jamile-dev** [comment]
É muita gente foda. Vou colocar o Twitter das pessoas da minha lista aqui porque é a rede social que mais acompanho. 😄 

- [Alda Rocha](https://twitter.com/mjcoffeeholick) 
- [Geisa Santos](https://twitter.com/geisasantos)
- [Tomomi Imura](https://twitter.com/girlie_mac)
- [Julia Evans](https://twitter.com/b0rk)
- [Mariko Kosaka](https://twitter.com/kosamari)
- [Wendy Ginsberg](https://twitter.com/wmginsberg)
- [Cynthia Zanoni](https://twitter.com/cynthiazanoni)
- [Carolina Pascale](https://twitter.com/CarolinaPascale)
- [Evelyn Mendes](https://twitter.com/jelielmendes)
- [Isa Silveira](https://twitter.com/silveira_bells)
- [Aline Bastos](https://twitter.com/AlineBastos)
- [Simone Amorim](https://twitter.com/samorim02)
- [Una Kravets](https://twitter.com/Una)
- [Loiane Groner](https://twitter.com/loiane)

**Comment by alinebastos** [comment]
Da mesma forma que a @JamileLima <3 eu acompanho basicamente o Twitter.
Além dessas pessoas fodas que já foram citadas aqui na issue, eu sigo esta lista maravilhosa:
https://twitter.com/AlineBastos/lists/devas/members
Consequentemente acompanho os posts no Medium dessas pessoas todas tb.

**Comment by shadowmaru** [comment]
Vixe, a lista é grande (e eu tirei muitos de backend porque ia ficar maior ainda):

* [Eduardo Matos](https://twitter.com/eduardojmatos)
* [Isa Silveira](https://twitter.com/silveira_bells)
* [Ramon Sanches](https://twitter.com/raymonsanches)
* [Iana Chan](https://twitter.com/iana)
* [Patricia Bastos](https://twitter.com/patricialuzbast)
* [Evelyn Mendes](https://twitter.com/jelielmendes)
* [Flora Saramago](https://twitter.com/flora_saramago)
* [Danilo Vitoriano](https://twitter.com/dnvtrn)
* [Felipe Fialho](https://twitter.com/LFeh)
* [Sandi Metz](https://twitter.com/sandimetz)
* [Katrina Owen](https://twitter.com/kytrinyx)
* [Paulo Silveira](https://twitter.com/paulo_caelum)
* [Anna Cruz](https://twitter.com/yuizinha)
* [Mattias Johansson (MPJ)](https://twitter.com/mpjme)
* [Helen Holmes](https://twitter.com/helenvholmes)
* [Talita Pagani](https://twitter.com/talitapagani)
* [Sergio Lopes](https://twitter.com/sergio_caelum)
* [Roberta Arcoverde](https://twitter.com/rla4)
* [Wes Bos](https://twitter.com/wesbos)
* [Lea Verou](https://twitter.com/LeaVerou)
* [Coraline Ada](https://twitter.com/CoralineAda)
* [Elvis](https://twitter.com/elvisdetona)
* [Vanessa Me Tonini](https://twitter.com/vanessametonini)
* [Raphael Fabeni](https://twitter.com/raphaelfabeni)
* [Leonardo Konarzewski](https://twitter.com/leonardokzw)
* [Jaydson Gomes](https://twitter.com/jaydson)
* [Rafael Rinaldi](https://twitter.com/rafaelrinaldi)
* [Miller Medeiros](https://twitter.com/millermedeiros)
* [Igor Apa](https://twitter.com/igorapa)
* [Almir Filho](https://twitter.com/almirfilho)
* [Jean Carlo Emer](https://twitter.com/jcemer)
* [Daniel Filho](https://twitter.com/danielfilho)
* [Zeno Rocha](https://twitter.com/zenorocha)
* [Lucas Mazza](https://twitter.com/lucasmazza)
* [Addy Osmani](https://twitter.com/addyosmani)
* [Paul Irish](https://twitter.com/paul_irish)
* [Carlos Souza](https://twitter.com/caike)
* [Guilherme Silveira](https://twitter.com/guilhermecaelum)
* [Nando Vieira](https://twitter.com/fnando)
* [Alda Rocha](https://twitter.com/mjcoffeeholick)
* E finalmente meu irmão [Eduardo Shiota](https://twitter.com/shiota)

**Comment by MathCunhaDev** [comment]
Vocês são fodas, pessoal!

Vlw pelas indicações. <3

**Comment by eduardojmatos** [comment]
@mariojvr @shadowmaru seus lindos! :)

Eu sigo quase todos que vcs comentaram aqui tbm :)

**Comment by mariojvr** [comment]
@eduardojmatos  kkkkkkkkkkkk <3

**Comment by talitapagani** [comment]
Que honra estar nessas listas @shadowmaru e @alinebastos! Sigo praticamente todos que foram indicados nesse post. Vou sugerir alguns canais/perfis do Medium que eu acompanho:

* [Alda Rocha](https://medium.com/@mjcoffeeholick)
* [Acessibílito](https://medium.com/@acessibilito)
* [Evelyn Jeliel Mendes](https://medium.com/@jelielmendes)
* [Reinaldo Ferraz](https://medium.com/@reinaldoferraz)
* [Tabless](https://medium.com/@tableless) (não poderia faltar)
* [Leandro Lima](https://medium.com/@leandro.lima)
* [UX Planet](https://medium.com/@uxplanet)
* [Adam Silver](https://medium.com/@adambsilver)
* [Simone Amorim](https://medium.com/@simoneas02)
* [Andrei Gurgel](https://medium.com/@andreigurgel)
* [Vandelay Design](https://medium.com/@VandelayDesign)
* [WebdesignerDepot](https://medium.com/@WebdesignerDepot)

**Comment by bella-silveira** [comment]
Muito honrada de ter sido citada, pessoal, muito obrigada @JamileLima @alinebastos e @shadowmaru ❤️
Sigo quase todo mundo já citado aqui também, só incluindo 

* [André Staltz](https://twitter.com/andrestaltz)
* [Andre Garzia](https://twitter.com/soapdog)
*  e a fofíssima [Marina Limeira](https://twitter.com/marinalimeira_)

**Comment by ghost** [comment]
Boa.
